# Crecimiento de la población en Jalisco
***

## Objetivos
---

> Encontrar la ecuación que determine la posición del cuerpo analizado respecto al tiempo.

> Emplear las funciones de python para resolver la ecuación diferencial presentada.

> Comparar la solución de la ecuación obtenida por medio de la programación con la solución analítica proporcionada.

> Interpretar la solución visual por medio de la teoría vista en clase y el marco teórico general de la física.

## Problema
---

> Un peso que pesa $64 lb$ se suelta desde una altura de $100 pies$ con una velocidad inicial de $10\frac{pies}{seg}$. Suponga que la resistencia del aire es proporcional a la velocidad del cuerpo. Se sabe que la velocidad límite debe ser $128\frac{pies}{seg}$.

> a) Encontrar una expresión para la velovidad del cuerpo en un momento $t$.

> b) Una expresión para la posición en un momento $t$.

## Modelo
---

> El modelo que utilizaremos para resolver nuestro problema es el siguiente:

$$\frac{dv}{dt}=kv$$

>Donde $v(t)$ es la velocidad con respecto al tiempo $t$.

## Solución
---

> A partir del problema podemos obtener los siquientes datos:

$$w = 64lb$$

$$w=mg$$

> Con esto se deduce que:

$$mg=64lb$$

> Despejando $m$ obtenemos que:

$$m=2slug$$

> Por otro lado se sabe que $v_1=128pies/seg$, por lo que obtenemos la siguiente igualdad:

$$128=\frac{64}{k}$$

> Despejando $k$:

$$k=\frac{1}{2}$$

>Sustituyendo en el modelo matématico queda:

$$\frac{dv}{dt}+\frac{1}{4}v=32$$

> Para resolverla se debe de igualar a $0$ la ecuación de la siguiente manera:

$$\frac{dv}{dt}+\frac{1}{4}v-32=0$$

In [1]:
import sympy as sym
import numpy as np
import matplotlib.pyplot as plt

In [2]:
sym.var('t')
v = sym.Function('v')
v(t)

v(t)

In [5]:
# Se define la ecuación simbólica
eq = sym.Derivative(v(t),t) + (1/4)*v(t)-32
eq

0.25*v(t) + Derivative(v(t), t) - 32

In [6]:
# Se integra para encontrar la ecuación de velocidad con respecto a t
velocidad = sym.dsolve(eq,v(t))
velocidad

Eq(v(t), C1*exp(-0.25*t) + 128.0)

> La ecuación de velocidad con respecto a $t$ está dada por:

$$v(t)=C_1e^{-0.25t}+128$$

> Para encontrar la constante $C_1$ se despejará de la ecuación $v(t)$:

$$C_1=\frac{v(t)-128}{e^{-0.25t}}$$

> Sustituyendo en ella las condiciones inciciales dadas por el problema:

$$t=0$$

$$v=10\frac{pies}{seg}$$

> Lo que nos queda:

$$C_1=\frac{10-128}{e^{-0.25(0)}}$$

In [11]:
def vel(t,v):
    return (v-128)/np.exp(-0.25*t)

t=0
v0=10

C1=vel(t,v0)
C1

-118.0

> El resultado de la constante queda:

$$C_1=-118$$

> La ecuación de velocidad para cualquier momento $t$:

<span style = "color: red">$$v(t)=-118e^{-0.25t}+128$$</span>

> Para encontrar la ecuación de posición para cualquier momento $t$ tenemos que:

$$v=\frac{dx}{dt}$$

> Donde $x$ es el desplazamiento

> Se puede escribir como:

$$\frac{dx}{dt}=-118e^{-0.25t}+128$$

## Visualización de la solución
---

## Conclusiones
---

## Referencias
---